client/components/product-card/options.jsx

```diff
@@ -29,7 +33,14 @@ const ProductCardOptions = ( {
 	return (
 		<Fragment>
 			{ ! hideRadios && optionsLabel && (
-				<h4 className="product-card__options-label">{ optionsLabel }</h4>
+				<h4 className="product-card__options-label">
+					{ optionsLabel }
+					<InfoPopover position="right">
+						{ translate(
+							'Records are all posts, pages, custom post types and other types of content indexed by Jetpack Search.'
```

Comment:
Added an oxford comma:

```suggestion
							'Records are all posts, pages, custom post types, and other types of content indexed by Jetpack Search.'
```

Reply:
Thanks!